src/test/java/net/technolords/micro/command/StatsCommandTest.java

```diff
@@ -18,11 +23,13 @@
 import net.technolords.micro.config.ConfigurationManager;
 import net.technolords.micro.registry.MockRegistry;
 
-public class StatsCommandTest {
+public class StatsCommandTest extends RouteTestSupport {
     private static final Logger LOGGER = LoggerFactory.getLogger(StatsCommandTest.class);
     private Main main;
     private ProducerTemplate producerTemplate;
     private String availablePort;
```

Comment:
While trying to move the logic of available port and setUp to Class Route Test Support, I get NPE. I see in debug mode, my reference object value is null. Test Class is extending RouteTestSupport, but in debug I see TestClassObject reference is not receiving the values from Parent Class.